x/bucket/types/bucket.go

```diff
@@ -0,0 +1,78 @@
+package types
+
+import (
+	fmt "fmt"
+	io "io"
+	"strings"
+
+	"github.com/sonr-io/sonr/pkg/did"
+	"github.com/sonr-io/sonr/pkg/did/ssi"
+)
+
+func (b *Bucket) GetPath(address string, segments ...string) string {
+	path := fmt.Sprintf("/%s/%s", address, b.Uuid)
```

Comment:
it seems we are creating a top level directory structure of `address/bucket-id` does this get created when the bucket is created?

Reply:
Yes see generate bucket @joshLong145